public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -28,6 +28,7 @@ export enum AlertmanagerAction {
   ViewNotificationTemplate = 'view-notification-template',
   UpdateNotificationTemplate = 'edit-notification-template',
   DeleteNotificationTemplate = 'delete-notification-template',
+  DecryptSecrets = 'can-read-secrets',
```

Comment:
nit: Can we update the string value here too?

Reply:
💯